File: src/main/kotlin/com/practiceKotlin/lec06/Lec06Main.java
package com.practiceKotlin.lec06;

import java.util.Arrays;
import java.util.List;

public class Lec06Main {
    public static void main(String[] args) {
        List<Long> numbers = Arrays.asList(1L, 2L, 3L);
        for (Long number : numbers) {
            System.out.println(number);
        }

        //1씩증가
        for(int i=1; i<=3; i++){
            System.out.println(i);
        }

        //1씩감소
        for(int i=3; i>=1; i--){
            System.out.println(i);
        }

        //2씩증가
        for(int i=1; i<=5; i+=2){
            System.out.println(i);
        }

    }
}

File: src/main/kotlin/com/practiceKotlin/lec06/Lec06Main.kt
package com.practiceKotlin.lec06

fun main(){
    val numbers = listOf(1L, 2L, 3L)
    for(number in numbers){
        println(number)
    }

    //1씩증가
    for(i in 1..3){
        println(i)
    }

    //1씩 감소
    for(i in 3 downTo 1){
        println(i)
    }

    //2씩증가
    for(i in 1..5 step 2){
        println(i)
    }

}